exporters/zipkin/src/zipkin_exporter.cc

```diff
@@ -0,0 +1,97 @@
+#include "opentelemetry/exporters/zipkin/zipkin_exporter.h"
+#include "opentelemetry/exporters/zipkin/recordable.h"
+#include "opentelemetry/ext/http/client/http_client_factory.h"
+#include "opentelemetry/ext/http/common/url_parser.h"
+
+OPENTELEMETRY_BEGIN_NAMESPACE
+namespace exporter
+{
+namespace zipkin
+{
+
+// -------------------------------- Contructors --------------------------------
+
+ZipkinExporter::ZipkinExporter(const ZipkinExporterOptions &options)
+    : options_(options), url_parser_(options_.endpoint)
+{
+  http_session_manager_ = ext::http::client::HttpClientFactory::Create();
+  InitializeLocalEndpoint();
+}
+
+ZipkinExporter::ZipkinExporter() : options_(ZipkinExporterOptions()), url_parser_(options_.endpoint)
+{
+  http_session_manager_ = ext::http::client::HttpClientFactory::Create();
+  InitializeLocalEndpoint();
+}
+
+// ----------------------------- Exporter methods ------------------------------
+
+std::unique_ptr<sdk::trace::Recordable> ZipkinExporter::MakeRecordable() noexcept
+{
+  return std::unique_ptr<sdk::trace::Recordable>(new Recordable);
+}
+
+sdk::trace::ExportResult ZipkinExporter::Export(
+    const nostd::span<std::unique_ptr<sdk::trace::Recordable>> &spans) noexcept
+{
+  if (isShutdown_)
+  {
+    return sdk::trace::ExportResult::kFailure;
+  }
+  auto session = http_session_manager_->CreateSession(url_parser_.host_, url_parser_.port_);
```

Comment:
Why do we create a new session for each call to `Export`? I see sessions are stored in the session manager. When are those sessions cleaned up? Will this cause an implicit memory leak (all past sessions being kept in the session manager)?

Reply:
`Session` object is confusing, as there is no actual session created for HTTP ( being stateless). I have simplified the http client library as part of #448 , and using that here:

```
  HttpClientSync httpClient;
  auto result = httpClient.Post(url, data, headers); // GET request
  if (result){
    auto response = result.GetResponse();
  } else {
    std::cout << result.GetSessionState();
  }
```